src/main/java/com/crud/tasks/sheduler/EmailScheduler.java

```diff
@@ -0,0 +1,34 @@
+package com.crud.tasks.sheduler;
+
+import com.crud.tasks.config.AdminConfig;
+import com.crud.tasks.domain.Mail;
+import com.crud.tasks.repository.TaskRepository;
+import com.crud.tasks.service.SimpleEmailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailScheduler {
+    private static final String SUBJECT = "Tasks: Once a day email";
+
+    @Autowired
+    private SimpleEmailService emailService;
+
+    @Autowired
+    private TaskRepository taskRepository;
+
+    @Autowired
+    private AdminConfig adminConfig;
+
+    //@Scheduled(cron = "0 0 10 * * *")
+    //@Scheduled(fixedDelay = 10000)
+    public void sendInformationEmail() {
+        long size = taskRepository.count();
+        emailService.send(new Mail(
+                adminConfig.getAdminMail(),
+                "adam.x.sikora@gmail.com",
+                SUBJECT,
+                String.format("Currently in database you got: " + size + (size == 1 ? "task" : "tasks"))));
```

Comment:
wyciagnij prosze zmienne do stalych. Task Stringi, jak i integery

Reply:
np _private static final int ONE_TASK = 1_

Reply:
poprawiłem co zauważyłem, jeśli jest jeszcze coś to daj znać